setup.cfg

```diff
@@ -19,7 +19,7 @@ console_scripts =
 max-line-length = 89
 max-complexity = 18
 select = B,C,E,F,W,T4,B9,B950
-ignore = W503, B903, E402, F541, B028, B907
```

Comment:
Better to understand the issue, and try to implement the correct behavior instead of the skipping rule...

By the way, please increase module service version if you making changes.